packages/design-system-react-native/package.json

```diff
@@ -92,8 +90,6 @@
     "@metamask/design-system-twrnc-preset": "^0.2.0",
     "@metamask/design-tokens": "^8.1.0",
     "@metamask/utils": "^11.8.0",
-    "@solana/addresses": "^2.0.0",
-    "bitcoin-address-validation": ">=2.0.0",
```

Comment:
These were removed in https://github.com/MetaMask/metamask-design-system/pull/816 but dependencies weren't updated